sample/kotlin/src/main/kotlin/com/datadog/android/sample/crash/CrashFragment.kt

```diff
@@ -62,11 +62,6 @@ class CrashFragment :
         viewModel = ViewModelProviders.of(this).get(CrashViewModel::class.java)
     }
 
-    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
-        super.onViewCreated(view, savedInstanceState)
-        triggerCrash()
```

Comment:
Is this crash mechanism no longer needed?

Reply:
Not when the view is created, we have buttons in the sample app ui to trigger different crashes (JVM, native,…)